storybook/src/stories/Filters/molecules/DatePicker.stories.tsx

```diff
@@ -17,34 +18,42 @@ export default {
   }
 };
 
-const exampleCallback =  <T extends Function>(fn: T): T => {
+const exampleCallback = <T extends Function>(fn: T): T => {
   /** A toString to render the function in storybook */
   // eslint-disable-next-line no-param-reassign
   fn.toString = () => '   updateCallback: (data: DateInterval | Date) => {}';
   return fn;
 };
 
 export const _DatePicker = () => {
-
+  const language = select("Language", { English: 'english', Japanese: 'japanese' }, "japanese");
```

Comment:
In locale translation we have english as en and japanese as ja so please keep value consistent with en and ja.

Reply:
Done